UITests/ViewMemoryLeakTests.swift

```diff
@@ -26,14 +26,18 @@ class ViewMemoryLeakTests: KIFTestCase, UITextFieldDelegate {
 
     func testAboutHomeDisposed() {
         // about:home is already active on startup; grab a reference to it.
-        let browserViewController = UIApplication.sharedApplication().keyWindow!.rootViewController!
+        let browserViewController = getTopViewController()
         weak var aboutHomeController = getChildViewController(browserViewController, childClass: "HomePanelViewController")
 
-        // Change the page to make about:home go away.
-        tester().tapViewWithAccessibilityIdentifier("url")
-        let url = "\(webRoot)/numberedPage.html?page=1"
-        tester().clearTextFromAndThenEnterTextIntoCurrentFirstResponder("\(url)\n")
-        tester().waitForWebViewElementWithAccessibilityLabel("Page 1")
+        // We have to use an autorelease pool here or else the weak reference to the about:home controller won't get cleared
+        // right away although the strong reference on the browser controller is set to nil.
+        autoreleasepool {
```

Comment:
You're testing if webView is cleaned up by the rest of the codebase, but not prevent that clean up happening simply by observing it in the test.

Could this use a weak ref to hold `webView` instead of using an autorelease pool?